supportedBackends/htcondor/src/test/scala/cromwell/backend/impl/htcondor/HtCondorCommandSpec.scala

```diff
@@ -5,16 +5,17 @@ import better.files._
 import org.scalatest.{Matchers, WordSpecLike}
 
 class HtCondorCommandSpec extends WordSpecLike with Matchers {
-  val attributes = Map("executable" -> "test.sh", "input" -> "/temp/test", "error"->"stderr")
-  val resultAttributes = List("executable=test.sh","input=/temp/test","error=stderr", "queue")
-  val htCondorCommands = new HtCondorCommands
+  private val attributes = Map("executable" -> "test.sh", "input" -> "/temp/test", "error"->"stderr")
+  private val resultAttributes = List("executable=test.sh","input=/temp/test","error=stderr", "spec1", "spec2", "queue")
+  private val htCondorCommands = new HtCondorCommands
+  private val nativeSpecs = Option(Array("spec1", "spec2"))
 
   "submitCommand method" should {
     "return submit file with content passed to it" in {
       val dir = File.newTemporaryFile()
-      val command = htCondorCommands.generateSubmitFile(dir.path,attributes)
+      val command = htCondorCommands.generateSubmitFile(dir.path, attributes, nativeSpecs)
       val file = dir
```

Comment:
What's this dance between `file` and `dir` for? Can't you just make `val file = File.newTemporaryFile()` in the first place?